containers/Ideas/IdeaCreate.js

```diff
@@ -122,7 +122,15 @@ export const IdeaCreate = props => {
           });
         }
         else if (statusCode == 400) {
-          setError('Required fields are missing.');
+          if (data && data.image) {
+            const errorString = data.image.reduce((acc, curr) => {
+              return `${acc}\n${curr}`;
+            }, 'Please try again.\n');
```

Comment:
is this error message meant to convay that the image is the wrong size? maybe check the error messages in a+

Reply:
shown along with backend message